bundles/org.openhab.binding.xmppclient/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="xmppclient"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="xmppBridge" extensible="publishTrigger">
+		<label>XMPP Client</label>
+		<description>A connection to a XMPP server</description>
+		<config-description>
+			<parameter name="username" type="text" required="true">
+				<label>Username</label>
+				<description>The XMPP Username (the left side of JID, e.g. user for JID user@example.com)</description>
+			</parameter>
+			<parameter name="domain" type="text" required="true">
+				<label>Domain</label>
+				<description>The XMPP Domain (the right side of JID, e.g. example.com for JID user@example.com)</description>
+				<context>network-address</context>
+			</parameter>
+			<parameter name="password" type="text" required="true">
+				<label>Password</label>
+				<description>The XMPP Password</description>
+				<context>password</context>
+			</parameter>
+			<parameter name="resource" type="text">
+				<label>Resource</label>
+				<description>The XMPP Resource name (if not specified, the openHAB will be used)</description>
```

Comment:
As far as I can tell, this description is no longer correct: This configured the name of the XEP-0030 identity. Most users would probably be happy if this is just set to "openHAB", there is possibly not even a need to expose it as user configurable knob.